threat-model.md

```diff
@@ -86,7 +86,12 @@ There is no risk in altering the ledger or performing double-spend attacks. Ther
 
 There are no risks of driving other aggregators out of business by denying them the chance to win rollups since they will get the reward of being active nodes.
 
+### Catastrophic events
+One crucial challenge of such a system is ensuring that some catastrophic event cannot leave all the value locked.
+[todo tudor]
```

Comment:
A lingering todo

Reply:
yeah, I need to write that when I get to that section